### DISCUSSION TITLE
Moving from HTTP to HTTPS

### DISCUSSION THREAD

**Discussion by khat17** [inventree/InvenTree#7802] [Q&A]
### Body of the issue

Hi there. Setup was originally done using:

https://docs.inventree.org/en/stable/start/installer/#script-options

Process included:
-------------------------------------------------------------------------------------------
sudo apt install software-properties-common

sudo add-apt-repository ppa:deadsnakes/ppa

sudo apt install python3.9

sudo wget -qO install.sh https://get.inventree.org && bash install.sh
-------------------------------------------------------------------------------------------

Above was used as no Docker instructions were clear enough (for me) to get it done.

Site is accessed using http://inventree.mysite.com with no issues. Would like to move from HTTP to HTTPS. Not seeing documentation on how to get this done.

Attempts to find .env file under the following paths not successful.

/opt/inventree/env
/opt/inventree/contrib/container

Not a power user or fluent in *nix. Just able to follow instructions and hobble along. Hopefully something can be provided to assist. Thanks in advance.

**Comment by matmair** [comment]
This is a question, not a documentation issue. I have moved this to discussions.

**Reply by khat17** [reply]
Thanks for that. Apologies for wrong section.

**Comment by szmolnar** [comment]
> ### Body of the issue
> Hi there. Setup was originally done using:
> 
> https://docs.inventree.org/en/stable/start/installer/#script-options
> 
> ## Process included:
> sudo apt install software-properties-common
> 
> sudo add-apt-repository ppa:deadsnakes/ppa
> 
> sudo apt install python3.9
> 
> ## sudo wget -qO install.sh https://get.inventree.org && bash install.sh
> Above was used as no Docker instructions were clear enough (for me) to get it done.
> 
> Site is accessed using http://inventree.mysite.com with no issues. Would like to move from HTTP to HTTPS. Not seeing documentation on how to get this done.
> 
> Attempts to find .env file under the following paths not successful.
> 
> /opt/inventree/env /opt/inventree/contrib/container
> 
> Not a power user or fluent in *nix. Just able to follow instructions and hobble along. Hopefully something can be provided to assist. Thanks in advance.

Me too I would like to use HTTPS.
Have you got an answer to this?

**Reply by khat17** [reply]
Nothing yet.
